rules/core/color-no-invalid-hex/README.md

```diff
@@ -0,0 +1,3 @@
+#### 📍 block-no-empty
```

Comment:
Change rule name

Reply:
I really shouldn't approve PRs while not feeling well

Reply:
ooopsy